File: OpenGUI-UnityScript/Scripts/OGRect.js
#pragma strict

@script AddComponentMenu ("OpenGUI/Rect")

class OGRect extends OGWidget {
	override function Draw ( x : float, y : float ) {	
		if ( !guiStyle ) { guiStyle = GUI.skin.box; }
			
		GUI.Box ( Rect ( x, y, transform.localScale.x, transform.localScale.y ), "", guiStyle );
	}
}

File: C#/Scripts/OGRect.cs
using UnityEngine;
using System.Collections;

[AddComponentMenu ("OpenGUI/Rect")]
public class OGRect : OGWidget {

	public override void Draw (  float x ,   float y   ){	
		if ( guiStyle == null) { guiStyle = GUI.skin.box; }
			
		GUI.Box ( new Rect( x, y, transform.localScale.x, transform.localScale.y ), "", guiStyle );
	}
}
